packages/libs/error-stack/tests/snapshots/test_debug__full__hook_fallback.snap

```diff
@@ -6,5 +6,13 @@ Root error
 ├╴tests/common.rs:147:5
 ├╴unknown
 ├╴unknown
-├╴unknown
-╰╴unknown
+├╴spantrace with 2 frames (1)
+╰╴backtrace with [n] frames (1)
+
+━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+Backtrace No. 1
+  [redacted]
+
+Span Trace No. 1
+  [redacted]
```

Comment:
This changed because the fallback is not overwritten anymore? I like that 🙂

Reply:
Yep c: